sdk/inc/azure/core/az_version.h

```diff
@@ -15,7 +15,7 @@
 #ifndef _az_VERSION_H
 #define _az_VERSION_H
 
-/// The version in string format used for telemetry following the semver.org standard
+/// The version in string format used for telemetry following the `semver.org` standard
```

Comment:
>  but I think we should verbatim the site-name.

Can you explain this to me. How was it not `verbatim` previously, or do you mean just add the back-ticks?

Reply:
Just adding the backticks, because "semver.org" is not a name itself, it is a place where the info can be found.